src/main/java/com/google/sps/servlets/TripsServlet.java

```diff
@@ -135,19 +138,25 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
    */
   public static Trip convertEntityToTrip(Entity entity) {
     String title = (String) entity.getProperty(Trip.ENTITY_PROPERTY_TITLE);
-    String hotel = (String) entity.getProperty(Trip.ENTITY_PROPERTY_HOTEL);
+    String hotelID = (String) entity.getProperty(Trip.ENTITY_PROPERTY_HOTEL_ID);
+    String hotelName = (String) entity.getProperty(Trip.ENTITY_PROPERTY_HOTEL_NAME);
+    String hotelImage = (String) entity.getProperty(Trip.ENTITY_PROPERTY_HOTEL_IMAGE);
     double rating = (double) entity.getProperty(Trip.ENTITY_PROPERTY_RATING);
     String description = (String) entity.getProperty(Trip.ENTITY_PROPERTY_DESCRIPTION);
     String owner = (String) entity.getProperty(Trip.ENTITY_PROPERTY_OWNER);
+    boolean isPastTrip = (boolean) entity.getProperty(Trip.ENTITY_PROPERTY_PAST_TRIP);
     boolean isPublic = (boolean) entity.getProperty(Trip.ENTITY_PROPERTY_PUBLIC);
```

Comment:
I talked to Charlie yesterday about possible scalability improvements, and he mentioned that enums are more scalable than booleans in case we decide to add more descriptive tiers rather than a simple true and false in the future. In this context, if I'm not mistaken, isPublic may have another middle tier where it can only viewed by followers, but this is just a suggestion to keep in mind for future properties.

Reply:
okay, that seems like a good idea. I don't think `enum` types will be very compatible with Datastore though, since [their types don't include them](https://cloud.google.com/appengine/docs/standard/python/datastore/entities#Properties_and_value_types). We would have to encode these enums into a string and then parse them when we query from the backend, which will result in a lot of extra boilerplate for a small design benefit IMO on the server-side. 

Perhaps we can just have `isPublic` be true if it's viewable by non-followers and then `isShared` as a secondary property that is true if viewable by followers only but then `isPublic` would be false.

Reply:
That'd be a good workaround, and my suggestion only assumes we may add a follower feature anyway.